README.md

```diff
@@ -329,8 +330,8 @@ smtpd_recipient_restrictions =
    reject_unknown_recipient_domain,
    permit_mynetworks,
    reject_unauth_destination,
-   reject_rbl_client zen.spamhaus.org,
-   reject_rbl_client bl.spamcop.net,
+   reject_rbl_client zen.spamhaus.org=127.0.0.[2..11],
```

Comment:
do you have any doc on spamhaus pointing to this change?

Reply:
I do not. Sorry, will revert.

Reply:
I agree with @hoyohayo : if using an open resolver almost every email would get blocked (https://www.spamhaus.org/news/article/788/spamhaus-dnsbl-return-codes-technical-update) and the standard return codes are [available here](https://www.spamhaus.org/faq/section/DNSBL%20Usage#200).

Reply:
I confirm en error when  using open resolvers, here are the error and spamhaus link:
554 5.7.1 Service unavailable; Client host [xxx] blocked using zen.spamhaus.org; Error: open resolver; https://www.spamhaus.org/returnc/pub/172.71.133.37;
As workaround I'm using google DNS for my dockers but I don't really like it

Reply:
@Metabaron1 I would recommend [something like this](https://www.linuxbabe.com/ubuntu/set-up-local-dns-resolver-ubuntu-20-04-bind9) alongside a PTR and you're good to go ;)